using_images/other_images/jenkins.adoc

```diff
@@ -386,7 +388,8 @@ Also note the `contrib` and `contrib/bin` subdirectories.  They allow for the in
 scripts for your image.
 
 The Jenkins image also provides auto-discovery and auto-configuration
-of slave images for the Kubernetes plug-ins by scanning the project Jenkins is deployed in for
+of slave images for the 
+Kubernetes plug-ins by scanning the project. Jenkins is deployed in for
```

Comment:
@gaurav-nelson your proposed change does not sound right to me.  How about this:

```
... of slave images for the Kubernetes plug-in.  The search for such images is done by examining existing image streams within the project that Jenkins is running in.  Specifically, the search looks for images streams that have the label *role* set to *jenkins-slave*.
```